abdulfatai360/banka
# #165479910 Migrate transactions endpoints API

#### What does this PR do?
Make the data of transactions endpoint API persistent

#### Description of Task to be completed?
- Have the APIs for transactions endpoints migrated
- Have the automated tests passed

#### How should this be manually tested?
On Terminal:
- Run `git clone https://github.com/abdulfatai360/banka.git` to clone this repo onto your local machine
- Navigate to the repo directory by running `cd banka/`
- Ensure your working directory is clean by running  `git status`  command
- Get a copy of the PR by typing  `git fetch origin pull/45/head:ch-migrate-transactions-endpoints-165479910`
- Now that you have a copy of the branch, switch to it using  `git checkout ch-migrate-transactions-endpoints-165479910`
- Your directory will now be an exact copy of the PR

#### What are the relevant pivotal tracker stories?
[#165479910](https://www.pivotaltracker.com/story/show/165479910)
